Formula/bareos-fd.rb

```diff
@@ -0,0 +1,72 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/17.2.4.tar.gz"
+  sha256 "4c443539012cf5ebb0fdb18878e604e82b951e6429c618acd18762f3c5724799"
+
+  depends_on "readline"
+  depends_on "openssl"
+
+  def install
+    system "./configure",
+    "--prefix=#{prefix}",
+    "--sbindir=#{bin}",
+    "--with-working-dir=#{var}/lib/bareos",
+    "--with-archivedir=#{var}/bareos",
+    "--with-confdir=#{etc}/bareos",
+    "--with-configtemplatedir=#{lib}/bareos/defaultconfigs",
+    "--with-scriptdir=#{lib}/bareos/scripts",
+    "--with-plugindir=#{lib}/bareos/plugins",
+    "--with-fd-password=XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX",
```

Comment:
This seems extremely weird to me. Is the default not sane? Ditto the ones below.

Reply:
There is no default, `XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX` is initialized and set by `deploy_config` as a random generated password in the `post_install` block. `--with-basename` and `--with-hostname` are catched from system and set to client name by `deploy_config`.

Reply:
What happens if we don't set these value in configure at all?

Reply:
`configure` will than generate a random password. For building locally, this is fine, but when bareos-fd is provided as pre-build package this is obviously a risk, as all clients would be accessible by the same password.

Reply:
Um … but doesn't this set a default password that has the same effect?

Reply:
No, because this is replaced in postinstall via `deploy_config` by a random password.

Reply:
This specific string is replaced and no other specific string would be?

Reply:
Exactly.

Reply:
Ah